### ISSUE TITLE
lbin missing

### ISSUE THREAD

**Issue by nerdralph** [WestfW/Minimal-ARM#1]
Readme says lbin contains linux binaries, but I see only wbin and macbin.

**Comment by WestfW** [comment]
Yeah.  I think I figured that linux users would be able to download and install the binaries from arm without needing as much hand-holding, and I got tired of trying the same thing on too many platforms.
These ARE just a subset of tools extracted from the downloads at https://launchpad.net/gcc-arm-embedded/
